sympy/polys/polyroots.py

```diff
@@ -605,7 +602,6 @@ def roots_quintic(f):
             break
 
     u, v = quintic.uv(theta, d)
-    sqrt5 = math.sqrt(5)
```

Comment:
this should be used, perhaps, in 609-610; perhaps only a low-precision value is needed there

Reply:
Yes, it could have been used there. My suspicion is that it was used there initially and then someone realized that it was nicer to get a symbolic root and forgot to remove it.

Another way would be to assign `sqrt5` to the symbolic variant, but it is not clear how much that actually saves (thinking that the original reason was to just evaluate the floating-point square root once).

Reply:
That was indeed added in a commit called "Minor changes for speed", https://github.com/sympy/sympy/commit/b7872957b4ea5dace4caa762f753ac83faee0a79 but only added, not used.